assets/scss/_home.scss

```diff
@@ -125,6 +125,33 @@
         bottom: 0;
         border-bottom: 4px solid #001E2A;
     }
+
+    .cabin {
+        position: absolute;
+        top: -2px;
+        width: max(903px, calc(100% + 5px));
+        height: 57vw;     
+        min-height: 400px;
+        max-height: 643px;
```

Comment:
At the "Desktop - Huge" size the Figma mockup shows the cabin image extending vertically to the `<div class="bottom-frame"></div>` separator at the bottom of the level. I think this `max-height` is causing a gap, as the image doesn't reach the separator.